### PR TITLE
BC-5414 - Add content element bar

### PR DESCRIPTION
**Pull request by bischofmax**
# Short Description

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/ui-board/content-element/ContentElementBar.vue [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+		<div
+			v-if="$slots.subtitle && ($slots.icon || $slots.title || $slots.menu)"
+			class="pt-0 pb-4 px-4"
+		>
+			<slot name="subtitle"></slot>
+		</div>
+
+		<div v-else-if="$slots.subtitle" class="pa-4">
+			<slot name="subtitle"></slot>
+		</div>
```
Comment (bischofmax) [inline src/components/ui-board/content-element/ContentElementBar.vue:40]: @SevenWaysDP Ich bin nicht zufrieden damit, dass wir hier Paddings handeln müssen. Hast du eine Idee?

File: src/components/feature-board-file-element/content/display/file-description/FileDescription.vue [hunk 2]
```diff
@@ -1,35 +1,34 @@
...
+			</template>
+		</ContentElementBar>
 
-			<slot></slot>
-		</v-app-bar>
-
-		<v-card-subtitle v-if="caption && !isEditMode">
-			{{ caption }}
-		</v-card-subtitle>
+		<div v-else-if="caption && !isEditMode" class="px-4 py-4 grey lighten-4">
```
Comment (bergatco) [inline src/components/feature-board-file-element/content/display/file-description/FileDescription.vue:23]: Both paddings can be combined to :
```suggestion
		<div v-else-if="caption && !isEditMode" class="pa-4 grey lighten-4">
```

File: src/components/feature-board-file-element/content/display/file-description/FileDescription.vue [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
-			</v-toolbar-title>
+			<template #menu>
+				<slot></slot>
+			</template>
 
-			<v-spacer></v-spacer>
+			<template #subtitle>
+				<span v-if="!isEditMode">
+					{{ caption }}
+				</span>
```
Comment (bergatco) [inline src/components/feature-board-file-element/content/display/file-description/FileDescription.vue:19]: This span (or at least its padding) is also shown, when caption is "empty".

image

**Review by bergatco — APPROVED** [review]

**Review by SevenWaysDP — APPROVED** [review]

File: src/components/feature-board-file-element/content/display/file-description/FileDescription.vue [hunk 4]
```diff
@@ -1,35 +1,34 @@
...
-					>{{ name }}</span
-				>
-			</v-toolbar-title>
+			<template #menu>
+				<slot></slot>
+			</template>
 
-			<v-spacer></v-spacer>
+			<template #subtitle>
+				<span v-if="caption && !isEditMode">
```
Comment (SevenWaysDP) [inline src/components/feature-board-file-element/content/display/file-description/FileDescription.vue:17]: Can you set v-if on template in line 16? without span?

File: src/components/feature-board-file-element/content/display/file-description/FileDescription.vue [hunk 5]
```diff
@@ -1,35 +1,34 @@
...
+		</ContentElementBar>
 
-			<slot></slot>
-		</v-app-bar>
-
-		<v-card-subtitle v-if="caption && !isEditMode">
-			{{ caption }}
-		</v-card-subtitle>
+		<div v-else-if="caption && !isEditMode" class="pa-4 grey lighten-4">
+			<span>{{ caption }}</span>
```
Comment (SevenWaysDP) [inline src/components/feature-board-file-element/content/display/file-description/FileDescription.vue:24]: Why do you need span?

**Review by SevenWaysDP — DISMISSED** [review]

**Review by SevenWaysDP — APPROVED** [review]

**Pull request merged**